docs/user-guide/framework-spec.md

```diff
@@ -128,11 +142,74 @@ There can be as many Custom modules as needed.
 
 TBD
 
-## Package Spec
-
-The `Package Spec` is based on the Framework spec `Types`, `Constraints`, and `Construct Types` defined above. It’s intended to serve as the interface between the high-level user-facing (App Devs) language layer and the low-level implementation layer. This leaks implementation hence not part of the framework spec at this point.
-
-### Package Level Attributes
+## Package Metadata
+
+The package metadata consist a pack of files covering all that has been mentioned above. It’s intended to be used by software tooling to understand the abstraction that a package makes available for project creators to use. The primary goal is making it a machine readable protocol so that software tooling can be developed to further aid project creators when using a selected package. 
+
+An example of the files is show below,
+
+![Package metadata files](../assets/cdf-package-definition-files.png)
```

Comment:
is there a reason why `metadata.ts` is not listed here?

Reply:
This is in a package where we expect `cdf.manifest.json` - `metadata.ts` is in CDF to help package authors load this manifest file, serving as a validator if you like